### PR TITLE
Remove single use and download links from a work drop down list in co…

### PR DESCRIPTION
**Pull request by tingtingjh**
fixes-1132_before
fixes-1132_after

### CODE REVIEW

File: app/views/collections/_show_document_list_menu.html.erb [hunk 1]
```diff
@@ -2,18 +2,12 @@
...
-                  class: "copypaste itemicon itemcode", title: "Single-Use Link to File", id: "copy_link_#{id}" %>
-    </li>
-    <li>
-      <%= link_to raw('<i class="glyphicon glyphicon-pencil"></i> Edit File'), sufia.edit_generic_file_path(id),
-                  class: 'itemicon itemedit', title: 'Edit File' %>
-    </li>
-    <li>
-      <%= link_to raw('<i class="glyphicon glyphicon-download-alt"></i> Download File'), sufia.download_path(id),
-                  class: 'itemicon itemdownload', title: 'Download File', target: '_new' %>
+      <%= link_to raw('<i class="glyphicon glyphicon-pencil"></i> Edit Work'), sufia.edit_generic_work_path(id),
```
Comment (cam156) [inline app/views/collections/_show_document_list_menu.html.erb]: @tingtingjh It would be really nice if the label was I18N instead of hard coded.  The wireframe show the label as just Edit (remove Work)

File: sufia-models/app/models/concerns/sufia/works/generic_work.rb [hunk 2]
```diff
@@ -5,7 +5,9 @@ module GenericWork
 
     include Sufia::Works::Work
     include Sufia::Works::CurationConcern::WithBasicMetadata
-    include Sufia::GenericFile::Permissions
+    include Sufia::GenericFile::Batches
```
Comment (cam156) [inline sufia-models/app/models/concerns/sufia/works/generic_work.rb]: Not sure why these changes are needed for work.  Is this just a quick fix for the collection size issue https://github.com/projecthydra/sufia/issues/1120?  If so my preferred method would be to change https://github.com/projecthydra/sufia/blob/master/sufia-models/app/models/concerns/sufia/collection_behavior.rb#L21 to just return zero.

File: app/views/collections/_show_document_list_menu.html.erb [hunk 3]
```diff
@@ -2,18 +2,12 @@
...
+                  class: 'itemicon itemedit', title: 'Edit Work' %>
```
Comment (cam156) [inline app/views/collections/_show_document_list_menu.html.erb]: If this title could also be i18N that would be fantastic.

File: sufia-models/app/models/concerns/sufia/works/generic_work.rb (hunk 2 above)
Comment (tingtingjh) [inline sufia-models/app/models/concerns/sufia/works/generic_work.rb]: Otherwise, for a work shown in collection work list, it will complain two methods (content and processing?) not found.

Comment (tingtingjh) [inline sufia-models/app/models/concerns/sufia/works/generic_work.rb]: These were added so that to make the Collection#show page work.

Comment (cam156) [inline sufia-models/app/models/concerns/sufia/works/generic_work.rb]: @tingtingjh As discussed on the call.  If you could just add a TODO to remove these times once the collection size and processing tickets are closed. #1120 & #1122 .  Thanks!

Comment (tingtingjh) [inline sufia-models/app/models/concerns/sufia/works/generic_work.rb]: Added TODO to remove these items once the collection size and processing tickets are closed. #1120 & #1122

**Comment by cam156** [comment]
:+1: waiting on travis

**Comment by cam156** [comment]
@tingtingjh  looks like you may need to rebase.

**Comment by tingtingjh** [comment]
Travis build has passed. Do I still need to rebase?

**Comment by cam156** [comment]
@tingtingjh yes.  The branch can not be merged via github.

**Comment by tingtingjh** [comment]
I did rebase and then merged with conflict fixed.

**Comment by cam156** [comment]
@tingtingjh can you please squash the commits?  One the commits are squashed it is good to go.
